src/System.Drawing.Common/src/misc/DebugHandleTracker.cs

```diff
@@ -429,7 +429,7 @@ private string GetLine()
                     }
 
                     /// <summary>
-                    /// Rereives the string of the parsed stack trace
+                    /// Retrieves the string of the parsed stack trace !!retrieves?
                     /// </summary>
                     public override string ToString()
                     {
```

Comment:
Sort of find the wording "retrieves" awkward with `.ToString()` here.

Reply:
I think it makes sense in the context of the type in question, especially that since it can *also* be considered a string representation of the object would be already known to anyone familiar enough with .NET to be digging into private classes in corefx.